README.md

```diff
@@ -23,10 +23,44 @@
 
 <p><img src="schedule.png" alt="LTS Schedule"/></p>
 
-The LTS Schedule is available also as a [JSON][] file or [ICal][]. There is
+The Release schedule is available also as a [JSON][] file or [ICal][]. There is
```

Comment:
I don't think we intended to enforce a schedule for Current releases, although there was some discussion in the LTS meeting that something like a 2 week interval might make sense.